thirdparty/hairer/radau5_c_py.pyx

```diff
@@ -214,20 +256,29 @@ cpdef radau5(fcn_PY, doublereal x, np.ndarray y,
 
     cdef integer idid = 1 ## "Successful compution"
     
-    iwork_in = np.array(iwork, dtype = np.int64)
+    iwork_in = np.array(iwork, dtype = np.int32)
     cdef np.ndarray[double, mode="c", ndim=1] y_vec = y
     cdef np.ndarray[double, mode="c", ndim=1] rtol_vec = rtol
     cdef np.ndarray[double, mode="c", ndim=1] atol_vec = atol
     cdef np.ndarray[double, mode="c", ndim=1] work_vec = work
     cdef np.ndarray[integer, mode="c", ndim=1] iwork_vec = iwork_in
+
+    if iwork[10]: ## sparse

```

Comment:
Maybe add a comment why `iwork[10]` is equivalent to sparse. Not necessary but to me it is very unclear.

Reply:
Changed it to 
`if iwork[10]: ## sparse linear solver flag, see radau_decsol_c.c`
with hopefully clarifies this.

the first part of the `iwork` array contains integer flags to select various solver options. These are all explained in the lengthy docstring in radau_decsol_c,c.